crates/matrix-sdk-base/src/client.rs

```diff
@@ -88,6 +88,7 @@ pub struct BaseClient {
     /// [`BaseClient::set_session_meta`]
     #[cfg(feature = "e2e-encryption")]
     olm_machine: OnceCell<OlmMachine>,
+    pub(crate) ignore_user_list_changes_tx: Arc<eyeball::shared::Observable<()>>,
```

Comment:
Can you add `use eyeball::shared::Observable;` next to the existing imports at the top of the file and use `SharedObservable<()>` here, for style consistency?

Reply:
done!